### ISSUE TITLE
hchart appears to be broken

### ISSUE THREAD

**Issue by abresler** [jbkunst/highcharter#568]
Looks like something related to rlang

```
data(diamonds, economics_long, mpg, package = "ggplot2")
library(dplyr)
library(highcharter)
hchart(mpg, "scatter", hcaes(x = displ, y = hwy, group = class))

```

**Comment by pachamaltese** [comment]
thanks @abresler for bringing this to our attention

here's the reprex, so I assigned this to myself

``` r
data(diamonds, economics_long, mpg, package = "ggplot2")
library(highcharter)
#> Registered S3 method overwritten by 'xts':
#>   method     from
#>   as.zoo.xts zoo
#> Registered S3 method overwritten by 'quantmod':
#>   method            from
#>   as.zoo.data.frame zoo
#> Highcharts (www.highcharts.com) is a Highsoft software product which is
#> not free for commercial and Governmental use
hchart(mpg, "scatter", hcaes(x = displ, y = hwy, group = class))
#> The quosure environment should be explicitly supplied as `env`
```

<sup>Created on 2019-10-27 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by pachamaltese** [comment]
@abresler ok, I was able to patch this for now
please run this `devtools::install_github("jbkunst/highcharter", ref = "720")`

**Comment by pachamaltese** [comment]
closed in 90cc6ff
